lib/use.js

```diff
@@ -0,0 +1,9 @@
+const { partial } = require('ramda')
+const { tapP }    = require('@articulate/funky')
+
+// use :: (IncomingMessage, HttpResponse, Function) -> Request -> Request
+const use = middle => tapP(({ request, response }) =>
+  new Promise(partial(middle, [ request, response ]))
+)
```

Comment:
I forgot about serverless. 🤦‍♂ 

Should we just throw if `request` is missing? I would hate to mislead someone to think that `use` was working...

Reply:
That's a good point.  Maybe throwing is better.  If we do throw, I wish we could do it earlier, at runtime, rather than waiting for the first request to come through.  But I'm not sure how to do that without a complete departure from the stateless nature of `paperplane`.

Reply:
I think that'd be fine.  I added a `warnOnce` to `gimme` to keep from flooding logs with warnings.  We could do something similar in this case if needed:
https://github.com/articulate/gimme/blob/master/lib/gimme.js#L27